tests/benchdnn/brgemm/brgemm.cpp

```diff
@@ -320,9 +339,12 @@ int doit(const prb_t *prb, res_t *res) {
     auto dst_md = dnn_mem_t::init_md(prb->ndims, prb->dst_dims.data(),
             prb->dst_dt(), prb->dtag, dst_strides);
 
-    using namespace dnnl::impl::cpu::x64;
-
+    using namespace namspace_impl;
+#if defined(brg_x64)
     brgemm_desc_t brgemm_desc;
+#else
+    brgemm_t brgemm_desc;
```

Comment:
Note: would be great if you could apply renaming for brgemm descriptor to avoid such branching. Doesn't have to be in this PR.

Reply:
Hi @dzarukin, thanks for the comment. Changing this would require changing the naming conventions throughout oneDNN. As suggested by you we would be including this renaming in a seperate PR.